src/Glpi/Migration/AbstractPluginMigration.php

```diff
@@ -59,7 +59,7 @@ abstract class AbstractPluginMigration
     /**
      * Current execution results.
      */
-    protected PluginMigrationResult $result;
+    public PluginMigrationResult $result;
```

Comment:
Please add a `getResult()` method, to prevent anyone to be able to redefine this property with another object.

```suggestion
    protected PluginMigrationResult $result;
```